source/addComment.js

```diff
@@ -1,3 +1,54 @@
 $(document).ready(function(){
-  // add your code here
+  $("#comments").on("click", "#new_comment_button", showCommentForm)
+
+  $("#comments").on("click", "[type=submit]", submitNewComment)
 })
+
+function showCommentForm() {
+  removeNewCommentButton()
+  createCommentForm()
+  unhideCommentForm()
+}
+
+function removeNewCommentButton() {
+  $("#new_comment_button").remove()
+}
+
+function createCommentForm() {
+  $("#comments").append(
+    "<form id=new_comment>" +
+    "<textarea placeholder='Your comment here...'></textarea>" +
```

Comment:
Indenting lines 20-22 will make it a little more readable; just like we would when nesting in an HTML document😊